lucien-ios/LoginScreen/RootViewController.swift

```diff
@@ -50,10 +56,29 @@ final class RootViewController: UIViewController, GIDSignInDelegate, GIDSignInUI
     func sign(_ signIn: GIDSignIn!, didSignInFor user: GIDGoogleUser!, withError error: Error!) {
         guard error == nil else { print("\(error.localizedDescription)"); return }
         guard let idToken = user.authentication.idToken else { return }
-        loginViewModel.authenticateUser(code: idToken)
+        loginViewModel.authenticateUser(code: idToken) {
+            self.loginViewModel.hasCollection {
+                print(self.loginViewModel.hasCollection)
+                self.loginViewModel.hasCollection ? self.showDashboard() : self.showStartMyCollection()
```

Comment:
This statement isn't quite right - we're in a callback from a function that should retrieve a Bool from the server. We shouldn't need to go into the view model to get that Bool, it should be part of the completion. Eventually we'll get rid of the completions entirely with Rx